EmailNotifications/TableStructure.cs

```diff
@@ -70,16 +70,41 @@ public void Dispose()
                 {
                     Append("</tr>");
                 }
-                public void AddCell(string innerText, string classAttributes = "", string id = "", string colSpan = "",string align ="", string  style= "", string fontSize = "")
+                public void AddCell(string innerText, string classAttributes = "", string id = "", string colSpan = "",string align ="", string  style= "", string fontSize = "",bool url = false)
                 {
                     Append("<td");
                     AddOptionalAttributes(classAttributes, id, colSpan, align, style); //put style value in here
+                    if (url == false)
+                    {
                     Append("<font");
-                    AppendOptionFont(classAttributes, id, fontSize,align);
+                    AppendOptionFont(classAttributes, id, fontSize, align);
                     Append(innerText);
                     Append("</font>");
+
+                    }
+                    else
+                    {
+                        Append("<a href="+innerText+">");
+                        Append("<font");
```

Comment:
The font element doesn't have this --> ">"

Reply:
Ryver me if you need further clarification